### PR TITLE
#304 use dictionary and log system from dynawo core

### LINKED ISSUE #304: Use dictionary system from dynawo

### PR DESCRIPTION
**Pull request by rosiereflo**
closes #304

### CODE REVIEW

**Comment by rosiereflo** [comment]
Let me fix the coverage

File: etc/Dictionaries/DFLLog_en_GB.dic [hunk 1]
```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2020, RTE (http://www.rte-france.com)
```
Comment (marcochiaramello) [inline etc/Dictionaries/DFLLog_en_GB.dic:1]: 2022

File: sources/Common/include/Log.h [hunk 2]
```diff
@@ -50,12 +54,22 @@ class Log {
...
-#define LOG(level) DYN::Trace::level(dfl::common::Log::dynaflowLauncherLogTag)
+#define LOG(level, key, ...)                                  \
+  DYN::Trace::level(dfl::common::Log::dynaflowLauncherLogTag) \
+      << (DYN::Message("DFLLOG", dfl::KeyLog_t::names(dfl::KeyLog_t::key)), ##__VA_ARGS__) << DYN::Trace::endline
 
 /**
- * @brief Alias of DYN::Trace::endline
+ * @brief Macro description to have a shortcut.
+ *  Thanks to this macro, user can only call an error with the type and the key to access
+ *  to the message (+ optional arguments if the message need)
```
Comment (marcochiaramello) [inline sources/Common/include/Log.h:66]: remove "to" before "the message"

**Review by marcochiaramello — CHANGES_REQUESTED** [review]

File: etc/Dictionaries/DFLLog_en_GB.dic (hunk 1 above)
Comment (rosiereflo) [reply etc/Dictionaries/DFLLog_en_GB.dic]: Done

File: sources/Common/include/Log.h (hunk 2 above)
Comment (rosiereflo) [reply sources/Common/include/Log.h]: Done

**Review by marcochiaramello — APPROVED** [review]

**Pull request merged**